src/entity/SearchChatMessage.java

```diff
@@ -0,0 +1,30 @@
+package entity;
+
+import java.time.LocalDateTime;
+
+public class SearchChatMessage implements ChatMessage {
+
+  private final LocalDateTime messageTime;
```

Comment:
I believe we should use Instant here instead of LocalDateTime because this is what is stored on the server (and use LocalDateTime for messages that are displayed to the user). Does that seem right?

Reply:
Sure